src/main/java/com/ays/user/model/dto/request/UserSupportStatusUpdateRequest.java

```diff
@@ -34,6 +38,9 @@ private boolean isSupportStatusValid() {
                 UserSupportStatus.ACCIDENT,
                 UserSupportStatus.OFFLINE
         );
+
         return EnumValidation.anyOf(this.supportStatus, acceptedUserSupportStatuses);
```

Comment:
Bu kısımda bir Set yapısı kullanmak yerine List yapısı kullanılabilir. 

Örneğin süreç içerisinde UserSupportStatus olarak isimlendirilmiş Enum yapısına yeni bir değer eklendiğinde veya çıkarıldığında dönüp burayı da manuel olarak güncellemek yerine `Arrays.stream(UserStatus.values()).toList()` kodunu kullanarak ilgili Enum yapısının içerisindeki değerleri içeren bir liste otomatik olarak oluşturulup kullanılabilir.

Elbette bu yapı kullanılmak istenirse EnumValidation isimli Utility sınıfının da anfOf metodunun ikinci parametresi olan `Set<T> acceptedValues` kısmının `List<T> acceptedValues` olarak güncellenmesi gerekecektir.

Örnek kullanım aşağıda verilmiştir.
`return EnumValidation.anyOf(this.status, Arrays.stream(UserStatus.values()).toList());`

Reply:
Burada sadece kullanıcın yapabileceği değerleri aldık. Gerisini admin tarafı yapacak. Durum bu şekilde.